ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,116 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+public class MarkPatientDeceasedPage extends Page {
+
+    private static final String URL = "/coreapps/markPatientDead.page";
+    private static final By DICTIONARY_LINK= By.id("dictionaryNavLink");
+    private static final By ADD_NEW_CONCEPT = By.linkText("Add new Concept");
+    private static final By CONCEPT_NAME = By.id("namesByLocale[en].name");
+    private static final By ADD_ANSWER_BUTTON = By.cssSelector("#codedDatatypeRow > td > table > tbody > tr > td.buttons > input:nth-child(1)");
+    private static final By CAUSE_BOX = By.id("newAnswerConcept");
+    private static final By FEVER = By.xpath("/html/body/ul[1]/li[2]/a");
+    private static final By ADD_BUTTON = By.xpath("/html/body/div[5]/div[11]/button[1]");
+    private static final By SAVE_CONCEPT_BUTTON = By.xpath("/html/body/div[1]/div[3]/div[2]/form/div/input[1]");
```

Comment:
Get better element reference for FEVER, ADD_BUTTON, and SAVE_CONCEPT_BUTTON rather than the xPath(). You can use` id()` or `cssSelector()`

Reply:
i have removed all xpaths and used eith cssselector or id